website/docs/components/form/super-select/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
+
+<Doc::Badge @type="warning">Conditionally Conformant</Doc::Badge>
+
+While many improvements have been made to the underlying library for this component, some variations, such as the multi-select, is still a work in progress and does not current pass the relevant WCAG Success Criteria. Please take this into consideration when implementing this component.
+
```

Comment:
@jorytindall @heatherlarsen I think we discussed that we should only use the terms "multiple" and "single" to match the component naming. Am I remembering correctly?

(If so I can update the text here.)

Reply:
Yes, I would expect "multiple" here.

Reply:
Agreed.

Reply:
Thanks, will update